schemas/jsonschema/Grid.json

```diff
@@ -0,0 +1,179 @@
+{
+  "$comment": "Generated from Grid by @foxglove/message-schemas",
+  "title": "Grid",
+  "description": "A 2D grid of data",
```

Comment:
Can this also be displayed in 3D space? (i.e in 3D panel)

Reply:
Yes, but the grid itself is 2D